crates/orchestrator/src/cli/storage/aws_s3.rs

```diff
@@ -2,7 +2,7 @@ use clap::Args;
 
 /// Parameters used to config AWS S3.
 #[derive(Debug, Clone, Args)]
-#[group()]
+#[group(requires_all = ["bucket_name"])]
```

Comment:
there was some reason why empty was fine, i remember asking @heemankv but can't recall now

Reply:
oh ya, it's because of the defalt value below. so it doesn't need a requires all